code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -158,9 +159,10 @@
 	suit_type = "hazmat hardsuit"
 	desc = "An Anomalous Material Interaction hardsuit that protects against the strangest energies the universe can throw at it."
 	icon_state = "science_rig"
-	armor = list(melee = 45, bullet = 5, laser = 45, energy = 80, bomb = 60, bio = 100, rad = 100)
+	armor = list(melee = 45, bullet = 5, laser = 40, energy = 65, bomb = 60, bio = 100, rad = 100)
 	slowdown = 1
 	offline_vision_restriction = 1
+	emp_protection = 40
```

Comment:
EMP protection on hardsuits actually seems to be pretty worthless. Check the EMP act in rig.dm, it doesn't seem to be factored in anywhere. It would probably be a good idea to do that.